pandas/core/series.py

```diff
@@ -887,21 +887,18 @@ def __getitem__(self, key):
         elif key_is_scalar:
             return self._get_value(key)
 
-        if (
-            isinstance(key, tuple)
-            and is_hashable(key)
-            and isinstance(self.index, MultiIndex)
-        ):
+        if is_hashable(key):
             # Otherwise index.get_value will raise InvalidIndexError
             try:
                 result = self._get_value(key)
 
                 return result
 
             except KeyError:
-                # We still have the corner case where this tuple is a key
-                #  in the first level of our MultiIndex
-                return self._get_values_tuple(key)
+                if isinstance(key, tuple) and isinstance(self.index, MultiIndex):
+                    # We still have the corner case where a tuple is a key
+                    # in the first level of our MultiIndex
+                    return self._get_values_tuple(key)
```

Comment:
would it make sense to handle the remaining hashable cases here instead of below?

Reply:
I think so, but perhaps that should be for 1.2? I am planning on doing a followup from @jreback's comment [here](https://github.com/pandas-dev/pandas/pull/36147#discussion_r484461769) and can include this too.

Reply:
yeah this is a bug fix, so can refactor in a followon.